.azure-pipelines/client.yml

```diff
@@ -173,11 +135,12 @@ jobs:
           codeCoverageTool: Cobertura
           summaryFileLocation: '$(Build.SourcesDirectory)/coverage.xml'
 
-  - job: Test_Alpha_Python
+# Run PyPy tests without coverage
+  - job: 'Test_PyPy'
 
-    timeoutInMinutes: 90
-    
-    condition: or(eq(variables['long_running_tests'], True), eq(variables['long_running_tests'], ''))
+    variables:
```

Comment:
Is there any reason this isn't another entry in the matrix strategy above?

Reply:
It seems the only difference is the extra disable coverage arg which could also be passed as an argument in the matrix strategy just for this entry and empty every where else.

Reply:
The reason is that performance for PyPy tests with coverage turned on is slow (over 1hr to finish the same tests which finish in 20 minutes for other). If we ran the `Test_PyPy` job as part of the `Test` job the coverage-related steps would fail and the pipeline would fail. 

Another approach we could take would be to set conditions on the coverage-related steps (which we're already doing with publishing to Code Cov). However this would involve bifurcating logic. 

Using `continueOnError: true` on the coverage steps results in the test job showing up as yellow in the pipeline.